services/frontend/src/components/Users/NewUserSection.jsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react'
+import { Form, FormGroup, Input, Button, Loader, Segment } from 'semantic-ui-react'
+import { useLazyGetUserFromSisuByEppnQuery, useAddUserMutation } from '@/redux/users'
+import { SortableTable } from '../SortableTable'
+
+export const NewUserSection = ({ onAddUser }) => {
+  const [eppn, setEppn] = useState('')
+  const [user, setUser] = useState('')
+  const [showAdded, setShowAdded] = useState(false)
+
+  const [
+    getUserFromSisuByEppnQuery,
+    { data: userFromApi, isLoading: isLoadingGetUser, isError: isErrorGetUser, isFetching },
+  ] = useLazyGetUserFromSisuByEppnQuery()
+  const [addUserMutation, { data: addedUser, isLoading: isLoadingAddUser, isError: isErrorAddUser }] =
+    useAddUserMutation()
+
+  useEffect(() => {
+    if (!isLoadingGetUser && !isErrorAddUser && !isFetching) {
+      setUser(userFromApi)
+    }
+  }, [userFromApi, isLoadingGetUser, isErrorGetUser, isFetching])
+
+  useEffect(() => {
+    if (!isLoadingAddUser && !isErrorAddUser) {
+      setUser('')
+      setEppn('')
+      onAddUser()
+      setShowAdded(true)
+      setTimeout(() => setShowAdded(false), 2000)
+    }
+  }, [addedUser, isLoadingAddUser, isErrorAddUser])
+
+  const handleEppnOnChange = (_, { value }) => {
+    if (value) setEppn(value)
+  }
+
+  const getUser = event => {
```

Comment:
Is it possible to call `getUser()` while `eppn` state = ´´? If so, is there a need to validate it before calling `getUserFromSisuByEppnQuery(eppn)`?

Most likely a non-issue

Reply:
Added an if (!eppn) return, just to be sure.